src/app/components/tissues-browser-grid-popover-content/tissues-browser-grid-popover-content.component.html

```diff
@@ -12,7 +12,9 @@
     </div>
 
     <div class="footer">
-      <mat-icon class="icon font-icon download" (click)="download()">cloud_download</mat-icon>
+      <a href="https://sampledata.hubmapconsortium.org/home" target="_blank">
```

Comment:
Shouldn't this point to link for the specific tissue?

Reply:
Also add rel="noreferrer noopener" to the <a> for extra security 😄

Reply:
the requirement is to have all the download button point to that link. @bherr2

Reply:
@KalaRaja for now it'll just link to the same site. We don't have a way to direct download the actual images right now.

Reply:
Ok